.pre-commit-config.yaml

```diff
@@ -26,11 +26,6 @@ repos:
         additional_dependencies: [tokenize-rt==3.2.0]
         exclude: _mars
         args: [--ignore-missing-imports, --follow-imports, skip]
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.0.0 # Use the sha or tag you want to point at
-    hooks:
-      - id: prettier
-        types_or: [html, javascript]
```

Comment:
Why remove this section? This is used for formatting web ui related codes. I think this has no relation with your codes.

Reply:
Yeah, I just remove it to make my code to pass the pre-commit. At last, I will return it back. Thank you.